packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -129,6 +133,8 @@ class Tabs extends React.Component {
     ) : null;
 
     const showScrollButtons = scrollable && (scrollButtons === 'auto' || scrollButtons === 'on');
+    const scrollButtonsActive =
+      scrollButtons === 'auto' && (this.state.showLeftScroll || this.state.showRightScroll);
```

Comment:
The fist check is redundant or rather shouldn't be part of the boolean variable.

Please destructure the state variables (should've been caught by a lint rule) to improve minification.

Reply:
Updated, should be good now